.env

```diff
@@ -0,0 +1,17 @@
+DEBUG=True
```

Comment:
Gitignore the `.env` file. Use a `sample.env` file, or even better - make the environment files local for the frontend and backend.